js/notify/electron-notify.js

```diff
@@ -711,6 +719,34 @@ function cleanUpInactiveWindow() {
     inactiveWindows = [];
 }
 
+/**
+ * Closes all the notifications and windows
+ */
+function closeAll() {
+    // Clear out animation Queue and close windows
+    animationQueue.clear();
+
+    activeNotifications.forEach(function(window) {
+        if (window.displayTimer) {
+            clearTimeout(window.displayTimer);
+        }
+        if (window.electronNotifyOnCloseFunc) {
+            // ToDo: fix this: shouldn't delete method on arg
```

Comment:
Should I wait for this to be fixed?

Reply:
Done. Removed the block of code as it is not required.